junifer/datagrabber/hcp.py

```diff
@@ -196,6 +199,7 @@ def __init__(
             datadir=datadir,
             tasks=tasks,
             phase_encodings=phase_encodings,
+            check_file_existence=check_file_existence,
```

Comment:
This does not need to be here. Only in the pattern one.

Reply:
Isn't it safe to keep it here as well considering it's the HCP datagrabber?

Reply:
Or maybe we explicitly pass `False` here?

Reply:
Okay on second thought, I will remove it, doesn't really harm.

Reply:
Done.